diff --git a/phylum/utils.lisp b/phylum/utils.lisp
index da3033c..5a9d22b 100644
--- a/phylum/utils.lisp
+++ b/phylum/utils.lisp
@@ -9,9 +9,15 @@
 ;; This block defines helper functions for account objects in the data model.
 ;; The helper functions have their own internal, private helpers which are
 ;; defined by labels.
-(labels ([make-account (acct-id balance)
+(labels ([make-account (acct-id balance &optional name)
           (sorted-map "account_id" acct-id
+                      "name" name
                       "current_balance" (to-int balance))]
+         [put-account-name! (acct-id name)
+          (statedb:put (format-string "{}.name" acct-id) name)
+          true]
+         [get-account-name (acct-id)
+          (statedb:get (format-string "{}.name" acct-id))]
          [get-balance (acct-id)
           (cc:infof (sorted-map "account_id" acct-id) "GET")
           (let* ([balance (statedb:get acct-id)])
