diff --git a/phylum/phylum.go b/phylum/phylum.go
index 590a4c2..577dfae 100644
--- a/phylum/phylum.go
+++ b/phylum/phylum.go
@@ -51,6 +51,9 @@ var (
 	phylumTransfer = &phylumMethod{
 		method: "transfer",
 	}
+	phylumDeleteAccount = &phylumMethod{
+		method: "delete_account",
+	}
 )
 
 func joinConfig(base []func() (Config, error), add []Config) (conf []Config, err error) {
@@ -288,3 +291,12 @@ func (s *Client) Transfer(ctx context.Context, req *pb.TransferRequest, config .
 	}
 	return resp, nil
 }
+
+func (s *Client) DeleteAccount(ctx context.Context, req *pb.DeleteAccountRequest, config ...Config) (*pb.DeleteAccountResponse, error) {
+	resp := &pb.DeleteAccountResponse{}
+	err := s.callMethod(ctx, phylumDeleteAccount, cmdParams(req), resp, config)
+	if err != nil {
+		return nil, err
+	}
+	return resp, nil
+}
